bot.py

```diff
@@ -143,6 +144,19 @@ async def invites(ctx, *args):
         await ctx.send(link)
 
 
+@bot.command()
+async def announce(ctx, channel:discord.TextChannel, *args):
+    await channel.send(f"📢 " + " ".join(str(x) for x in args) + " 📢")
```

Comment:
Please remove the 📢s in between the message. As the user could add them if needed.

Reply:
And the problem with this approach is that, if the user sends a properly formatted message with tabs and spaces, the message will be just joined with whitespaces. 

So if i send a message like this, 
<img width="786" alt="Screenshot 2021-10-03 at 11 54 59 AM" src="https://user-images.githubusercontent.com/22593204/135742647-bc443868-fc8f-444b-80cd-82b5212b6034.png">

The output will be this:
<img width="786" alt="Screenshot 2021-10-03 at 11 55 51 AM" src="https://user-images.githubusercontent.com/22593204/135742665-d006833d-334e-4892-89b1-dfeae6abb88e.png">

So figure out a method to even take in whitespaces and tabs and all!

Reply:
The new commit solves the issue

Reply:
Yes it does!